static/js/components/photometry/UpdatePhotometry.jsx

```diff
@@ -89,6 +86,40 @@ const UpdatePhotometry = ({ phot, magsys }) => {
     newState.filter = subState.filter;
     newState.magsys = magsys;
 
+    Object.keys(newState).forEach((key) => {
+      if (
+        newState[key] === null ||
+        newState[key] === undefined ||
+        newState[key] === ""
+      ) {
+        delete newState[key];
+      }
+    });
+
+    if (
+      newState?.limiting_mag === undefined &&
+      newState?.mag === undefined &&
```

Comment:
Does this actually require a magnitude and magnitude error? Should it?

Reply:
I mean I think this is the issue we are trying to fix right? We don't need a mag and magerr if we have a limiting mag, and vice versa. But we do need one of these 2 options (detection or non detection data) to be given.

Reply:
Ok great.